pkg/api/types.go

```diff
@@ -2121,6 +2121,16 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "system" is a special keyword
```

Comment:
Agreed. We should find a better naming convention for default priority class name(s). The choice of names won't affect this PR (except for the comment here) though.